sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/src/Listeners/EventHubListener.cs

```diff
@@ -153,22 +264,23 @@ public async Task ProcessEventsAsync(EventProcessorHostPartition context, IEnume
                 using (CancellationTokenSource linkedCts =
                         CancellationTokenSource.CreateLinkedTokenSource(_cts.Token, processingCancellationToken))
                 {
-                    var events = messages.ToArray();
+                    _mostRecentPartitionContext = context;
+                    var events = messages == null ? Array.Empty<EventData>() : messages.ToArray();
```

Comment:
An enumerable should never be null.   What happened that we need this?

Reply:
I was getting null exceptions from this line at one point, but was never able to track them down via the debugger (or figure out where they were coming from logically), I'll try testing it without it and see if I'm still getting that exception.